runtime/src/lib.rs

```diff
@@ -642,6 +630,49 @@ impl pallet_session::Config for Runtime {
 	type WeightInfo = ();
 }
 
+parameter_types! {
+	pub IntergalacticAccounts: Vec<AccountId> = vec![
+		hex_literal::hex!["0abad795adcb5dee45d29528005b1f78d55fc170844babde88df84016c6cd14d"].into(),
+	];
+}
+
+pub struct EnsureIntergalactic;
+impl EnsureOrigin<Origin> for EnsureIntergalactic {
+	type Success = AccountId;
+
+	fn try_origin(o: Origin) -> Result<Self::Success, Origin> {
+		Into::<Result<RawOrigin<AccountId>, Origin>>::into(o).and_then(|o| match o {
+			RawOrigin::Signed(caller) => {
+				if IntergalacticAccounts::get().contains(&caller) {
+					Ok(caller)
+				} else {
+					Err(Origin::from(Some(caller)))
+				}
+			}
+			r => Err(Origin::from(r)),
+		})
+	}
+
+	#[cfg(feature = "runtime-benchmarks")]
+	fn successful_origin() -> Origin {
+		Origin::from(RawOrigin::Signed(Default::default()))
+	}
+}
+
+parameter_types! {
+	pub MinVestedTransfer: Balance = BSX / 100_000;
```

Comment:
```suggestion
	pub MinVestedTransfer: Balance = BSX * 100_000;
```

Reply:
this is just the minimum for the vested_transfer call defined by the caller

Reply:
i assume this is safety mechanism to prevent spamming out limited number of vestings u can have at the sam etime

Reply:
yes, but depends, if only a privileged account can call vested_transfer it should be no problem?